crates/js-sys/src/lib.rs

```diff
@@ -1493,21 +1502,33 @@ impl fmt::Debug for BigInt {
 impl fmt::Display for BigInt {
     #[inline]
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.pad_integral(self >= &BigInt::from(0), "", &self.to_string_unchecked(10))
+        f.pad_integral(
+            self >= &BigInt::from(0),
+            "",
+            &self.abs().to_string_unchecked(10),
+        )
```

Comment:
The check `self >= &BigInt::from(0)` is being done twice now, probably the easiest thing here is to let `abs()` also return a `bool`.